modules/nf-core/ncbitools/vecscreen/tests/main.nf.test

```diff
@@ -0,0 +1,81 @@
+nextflow_process {
+
+    name "Test Process NCBITOOLS_VECSCREEN"
+    config "./nextflow.config"
+    script "../main.nf"
+    process "NCBITOOLS_VECSCREEN"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "blast"
+    tag "blast/makeblastdb"
+    tag "ncbitools"
+    tag "ncbitools/vecscreen"
+
+    setup {
+        run("BLAST_MAKEBLASTDB") {
+            script "../../../blast/makeblastdb/main.nf"
+            config './nextflow.config'
+            process {
+                """
+                input[0] = [[id:'test'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/pacbio/fasta/primers.fasta', checkIfExists: true)
+                ]
+                """
+            }
+        }
+    }
+
+    test("homo sapiens") {
+        options '-stub'
```

Comment:
@LouisLeNezet , this is why the file is empty.

Reply:
I didn't catch that. But why do you test twice the stub ?

Reply:
I didn't mean to!